application/models/Template.php

```diff
@@ -509,6 +509,29 @@ public static function resetInstance()
         self::$instance = null;
     }
 
+    /**
+     * Reset assets for this template
+     * Using a new file and not touch, seems some winows version didn't work with touch (?)
+     * @throw Exception in case we can not find a new random file name in 10 attempt
+     * @return void
+     */
+    public function resetAsset()
+    {
+        /* Add a test for directory writable ? Break before this is needed, then not need */
+        $templatePathWithDirSeparator = self::getTemplatePath($this->name).DIRECTORY_SEPARATOR;// Unsure directory separator is needed but … windows …
+        $fileToTouch= App()->securityManager->generateRandomString(42);
+        $counter = 0;
+        while (is_file($templatePathWithDirSeparator.$fileToTouch)) {
+            $counter++;
+            // This is extremely unlikely.
+            if ($counter > 10) {
+                throw new CHttpException(500, 'Failed to reset template asset with new file.');
+            }
+            $fileToTouch= App()->securityManager->generateRandomString(42);
+        }
+        touch($templatePathWithDirSeparator.$fileToTouch);
+        unlink($templatePathWithDirSeparator.$fileToTouch);
+    }
```

Comment:
No: we don't want to use any more the "touch"  logic. 
On 2.50+ we face the fact that the file system behave differently in many situation: SELinux, Linux, windows, specific config, etc 
So on LS3 the touch logic has been replaced by an Asset Version Number, one in the version file for the ComfortUpdate, the other one in DB for the user.